src/LogViewer/chunk-worker.js

```diff
@@ -6,7 +6,6 @@ const NORMALIZE_NEWLINES = /\r[\n]?/gm;
 const NEWLINE = /^/gm;
 const ENCODED_CARRIAGERETURN = 13;
 const ENCODED_NEWLINE = 10;
-const MIN_LINE_HEIGHT = 19;
```

Comment:
If you switch this to a `let`, you can override in the postMessage and not have to pass it around everywhere:

```js
let minLineHeight = 19;
```

```js
chunkHeights.push(LINE_CHUNK * minLineHeight);
...
chunkHeights.push(newlineCount * minLineHeight);
```

```js
switch (data.type) {
  case 'start':
    if (data.lineHeight) {
      minLineHeight = data.lineHeight;
    }
}
```